kotlinx-coroutines-debug/src/DebugProbes.kt

```diff
@@ -114,7 +121,7 @@ public object DebugProbes {
      * Returns all existing coroutines info.
      * The resulting collection represents a consistent snapshot of all existing coroutines at the moment of invocation.
      */
-    public fun dumpCoroutinesInfo(): List<CoroutineInfo> = DebugProbesImpl.dumpCoroutinesInfo()
+    public fun dumpCoroutinesInfo(): List<CoroutineInfo> = DebugProbesImpl.dumpCoroutinesInfo().map { CoroutineInfo(it) }
```

Comment:
The difference between:
* `dumpDebuggerInfo` -> returning `List<DebufferInfo>` and
* `dumpCoroutinesInfo` -> returning `List<DebugCoroutineInfo>`
it too subtle and the names do not help to figure out which does what.

Do we really need both?

Reply:
`dumpCoroutinesInfo` is publci a user-facing API.
`dumpDebuggerInfo` is an internal method for IDEA plugin that should return serializable list. I will properly document and test the latter as soon as I know that it fits IDEA needs

Reply:
Maybe we should make `dumpDebuggerInfo` internal then? (with clean `@JvmName`, of course)

Reply:
`dumpDebuggerInfo` lies in `DebugProbesImpl` which is already internal

Reply:
It is public in a  `public object DebugProbes`.

Reply:
I see. I was just confused by the names, then.